app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
```

Comment:
Simplify this condition since it is looking for th e presence of the string in the keys by default
```suggestion
        if "vaccine" not in visitor:
```